src/celeritas/ext/ScopedRootErrorHandler.cc

```diff
@@ -43,6 +43,8 @@ void RootErrorHandler(Int_t rootlevel,
     {
         level = LogLevel::error;
         g_has_root_errored_ = true;
+        if (strcmp(location, "TCling::LoadPCM") == 0) == 0)
```

Comment:
`std::strcmp` and include `<cstring>`. Or could this be made more legible with `corecel/io/StringUtils.cc`:
```suggestion
        if (starts_with(location, "TCling::LoadPCM"))
```

Reply:
The intent here is to find an exact match.